templates/parachain/zombienet/tests/smoke.rs

```diff
@@ -0,0 +1,49 @@
+//! This test is setup to run with the `native` provider and needs these binaries in your PATH
+//! `polkadot`, `polkadot-prepare-worker`, `polkadot-execute-worker`, `parachain-template-node`.
+//! You can follow these steps to compile and export the binaries:
+//! `cargo build --release -features fast-runtime --bin polkadot --bin polkadot-execute-worker --bin
+//! polkadot-prepare-worker`
+//! `cargo build --package parachain-template-node --release`
+//! `export PATH=<path-to-polkadot-sdk-repo>/target/release:$PATH
```

Comment:
```suggestion
```

Reply:
You still need to have `<path-to-polkadot-sdk-repo>/target/release` in your PATH in order to run `parachain-template-node`, `minimal-template-node`, etc.